ConnectorAutocadCivil/ConnectorAutocadCivil/Utils.cs

```diff
@@ -606,7 +702,7 @@ string fileNameHash
           }
         }
       }
-      if (foundObjects.Any())
+      if (foundObjects.Count != 0)
```

Comment:
I know this was pre-existing, but can we add a space between the end of one if block and this if statement plZ?